# HG changeset patch
# User Derek Schuff <dschuff@google.com>
# Date 1310167630 25200
# Branch pnacl-sfi
# Node ID d2d82032032b4c447d87b383e299cd998b83b360
# Parent 761fb6fa84084e96da90ea6c5875774b641af299
Fix link line for tcmalloc

 From pnacl-llvm-0052-277-d2d82032032b4c447d87b383e299cd998b83b360.patch

diff -r 761fb6fa8408 Makefile.rules
--- a/Makefile.rules	Fri Jul 08 13:36:53 2011 -0700
+++ b/Makefile.rules	Fri Jul 29 14:32:46 2011 -0700
@@ -580,8 +580,8 @@
 ifeq ($(NACL_SANDBOX),1)
   LIBS += -lsrpc -limc_syscalls -lplatform -lgio -lpthread -lm -lnacl
   ifeq ($(USE_TCMALLOC),1)
-    # Note: -ltcmalloc needs to stay last on the link line
-    LIBS += -ltcmalloc
+    # Note: -ltcmalloc_minimal needs to stay last on the link line
+    LIBS += -ltcmalloc_minimal
     CXX.Flags += -fno-builtin-malloc -fno-builtin-calloc -fno-builtin-realloc -fno-builtin-free
     C.Flags += -fno-builtin-malloc -fno-builtin-calloc -fno-builtin-realloc -fno-builtin-free
   endif
